store/mockstore/mocktikv/rpc.go

```diff
@@ -619,17 +619,20 @@ type RPCClient struct {
 	Cluster       *Cluster
 	MvccStore     MVCCStore
 	streamTimeout chan *tikvrpc.Lease
+	done          chan struct{}
 }
 
 // NewRPCClient creates an RPCClient.
 // Note that close the RPCClient may close the underlying MvccStore.
 func NewRPCClient(cluster *Cluster, mvccStore MVCCStore) *RPCClient {
-	ch := make(chan *tikvrpc.Lease)
-	go tikvrpc.CheckStreamTimeoutLoop(ch)
+	ch := make(chan *tikvrpc.Lease, 1024)
+	done := make(chan struct{})
+	go tikvrpc.CheckStreamTimeoutLoop(ch, done)
 	return &RPCClient{
 		Cluster:       cluster,
 		MvccStore:     mvccStore,
 		streamTimeout: ch,
+		done:          done,
```

Comment:
Why add the `done` channel?

Reply:
Will the `CheckStreamTimeoutLoop` goroutine exit?

Reply:
> Why add the `done` channel?

The old way would panic
1. Invoke `RPCClient.Close` to close channel `streamTimeout`
2. An inflight SenRequest send a message into channel `streamTimeout`